src/tribler/ui/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "tribler-webui",
-  "version": "0.1",
+  "version": "0.1.0",
```

Comment:
Question: are all these edits intended?

Reply:
Yep, if I do `npm i jslint` the version becomes `0.1.0` for some reason.